### PR TITLE
Docs: Mobile docs added for Tabs

### PR DESCRIPTION
**Pull request by ponori**
Added docs for Android/iOS Tabs component

### CODE REVIEW

File: docs/markdown/android/tabs.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Tabs
+description: Tabs may be used navigate between multiple URLs. Tabs are intended as page-level navigation.
```
Comment (ayeshakmaz) [inline docs/markdown/android/tabs.md:3]: typo: "Tabs may be used **to** navigate..."

File: docs/markdown/android/tabs.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/fe/97/b2/fe97b2a279741762771f98bcd47e8e15.png" alt="example of tabs with truncated labels"/>
+    <Dont title="Don't" />
+    Truncate tabs to fit within a view.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/09/e9/b6/09e9b68dea0327b92311472655e416a1.png" alt="example of using too many tabs"/>
+    <Dont title="Don't" />
+    Use more than 6 Tabs. Consider using a different component like a Sheet.
+  </Group>
```
Comment (ayeshakmaz) [inline docs/markdown/android/tabs.md:56]: given that this is already mentioned above in the bullet points, and it doesn't have a Do match, can we delete it?

**Review by ayeshakmaz — COMMENTED** [review]
Comments apply for both pages

File: docs/markdown/android/tabs.md (hunk 2 above)
Comment (ponori) [reply docs/markdown/android/tabs.md:56]: I'm down. Any concerns @khalvy?

Comment (khalvy) [reply docs/markdown/android/tabs.md:56]: Yea, sounds good to me!

Comment (ponori) [reply docs/markdown/android/tabs.md]: Done!

**Review by ayeshakmaz — APPROVED** [review]

**Pull request merged**